zebra-state/src/service.rs

```diff
@@ -287,7 +287,7 @@ impl StateService {
     ///
     /// The block identified by `hash` is included in the chain of blocks yielded
     /// by the iterator.
```

Comment:
Let's also update this comment to say "any chain".

```suggestion
    ///
    /// The block identified by `hash` is included in the chain of blocks yielded
    /// by the iterator. `hash` can come from any chain.
```